cmd/gen-user/user.go

```diff
@@ -0,0 +1,144 @@
+package main
+
+import (
+	"crypto/sha512"
+	"encoding/base32"
+	"encoding/base64"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/nats-io/jwt/v2"
+	"github.com/nats-io/nkeys"
+)
+
+func createUser(accSeed []byte, opts []Opt) (string, string, error) {
```

Comment:
Let's stay consistent and declare opts as variadic param.

Also, maybe it makes sense to extend `options.CreateUser` to accept custom user claims?

I'm asking, because right now `options.CreateUser` looks like becomes obsolete and is not removed.
However, `CreateUser` was created as a workaround and, frankly, was misplaced - it should not live in `options`, but be a part of a separate package dedicated to NATS credentials and other NATS specifics.

On the other hand, I think it is best to create a package that has various helpers related to NATS, move this code there, and `options.CreateUser` should be removed.

Reply:
Made opts variadic arg 👍 

Will think about extending `options.CreateUser`.

Reply:
Moved common part into `pkg/user`, so now it's `user.CreateCreds`.
Do you want me to remove `options.CreateUser`?

Reply:
On the other hand, maybe copy `options.CreateUser` to `user` package. IIRC Osmosis publisher uses it in one form or the other(I think it implements its own function, but it should migrate to `data-layer-sdk` implementation).